pkg/cmd/pulumi/util.go

```diff
@@ -500,6 +497,33 @@ func parseAndSaveConfigArray(s backend.Stack, configArray []string, path bool) e
 	return nil
 }
 
+// parseConfigArray parses the config array
+// returns empty config if configArray argument is empty
+func parseConfigArray(configArray []string, path bool) (config.Map, error) {
+	if len(configArray) == 0 {
```

Comment:
This check isn't needed. `parseConfig` correctly handles an empty array.

Reply:
@Frassle fixed